server/src/server.rs

```diff
@@ -137,6 +137,32 @@ pub async fn handle_put_messages<T: SignalDatabase, U: WSStream + Debug>(
     Ok(SendMessageResponse { needs_sync })
 }
 
+pub async fn handle_keepalive<T: SignalDatabase, U: WSStream + Debug>(
+    state: &SignalServerState<T, U>,
+    authenticated_device: &AuthenticatedDevice,
+) -> Result<(), ApiError> {
+    //Check if present in presencemanager. If not present, close connection for device. Else return 200 Ok
+    if !state
+        .client_presence_manager
+        .is_locally_present(&authenticated_device.get_protocol_address(ServiceIdKind::Aci))
+    {
+        if let Some(connection) = state
+            .websocket_manager
+            .get(&authenticated_device.get_protocol_address(ServiceIdKind::Aci))
+            .await
+        {
+            connection
+                .lock()
+                .await
+                .close_reason(1000, "OK")
+                .await
+                .map_err(|err| err.to_string());
```

Comment:
Add '?' to send error, otherwise it is not used

Reply:
Endpoints returnerer strings som error. Ikke min ide